README.md

```diff
@@ -45,6 +47,14 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). 
 
+Pip install Arrow for an easier setup with the testing of the application.  Arrow is a Python library that offers a sensible and human-friendly approach to creating, manipulating, formatting and converting dates, times and timestamps.
+
+```bash
+$ pip install -U arrow
+
+$ poetry add arrow 
```

Comment:
The reader shouldn't need to manually add arrow. When you added it, the project's list of dependencies was updated (see pyproject.toml).

So now `poetry install` will install arrow alongside other dependencies like Flask or requests.